cla_backend/apps/call_centre/sql/ExportLegalAidEligibilityCheck.sql

```diff
@@ -0,0 +1,23 @@
+COPY (SELECT
+id,
+created,
+modified,
+reference,
+category_id,
+you_id,
+partner_id,
+disputed_savings_id,
+your_problem_notes,
+notes,
```

Comment:
Isn't this potentially sensitive? If Yes, maybe we should delete this and eventlog.log.patch as well. What do you think @r4vi ?